Azure/sonic-buildimage
# [Mellanox] Add initial support for SN4800 platform

#### Why I did it
Add initial support of SN4800 platform . 
NOTE: This is work in progress and not full support of the platform. 

#### How I did it
Add new folders for SN4800 with zero ports based on SN4700 Spectrum-3 switch.

#### How to verify it
Simulator device was tested. See https://github.com/Azure/sonic-buildimage/pull/7448

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

new folder for SN4800 device
The sai_4800.xml & port_config.ini are with 0 ports
